dispatch/tests/test_api_auth.py

```diff
@@ -0,0 +1,111 @@
+from django.core.urlresolvers import reverse
+
+from rest_framework import status
+from rest_framework.authtoken.models import Token
+
+from dispatch.tests.cases import DispatchAPITestCase
+
+from dispatch.apps.content.models import Article, User
+
+class AuthenticationTests(DispatchAPITestCase):
+
+    def test_login(self):
+        """
+        login(email, password) -> user logs in successfully, token is returned by api
+        """
+        url = reverse('auth-token')
+
+        data = {
+            'email': 'test@test.com',
+            'password': 'testing123'
+        }
+
+        response = self.client.post(url, data, format='json')
+        self.assertIsNotNone(response.data.get("token"))
+        self.assertEqual(response.status_code, status.HTTP_202_ACCEPTED)
+
+    def test_logout(self):
+        """
+        login(email, password) -> logout -> user logs out successfully, db should no longer have token
+        """
+        url = reverse('auth-token')
+
+        logout_response = self.client.delete(url, {}, format='json')
+
+        # Try to get the token, if it doesn't exist, return None.
+        user = User.objects.get(email='test@test.com')
+        try:
+            token = Token.objects.get(user_id=user.id)
+        except Token.DoesNotExist:
+            token = None
+
+        self.assertIsNone(token)
```

Comment:
I think it might be cleaner to use [assertRaises](https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertRaises) here